akka-docs/src/main/paradox/general/stream/stream-design.md

```diff
@@ -45,6 +45,27 @@ All stream Processors produced by the default materialization of Akka Streams ar
 
 This means that `Sink.asPublisher(true)` (for enabling fan-out support) must be used where broadcast behavior is needed for interoperation with other Reactive Streams implementations.
 
+### Rationale and benefits from Sink/Source/Flow not directly extending Reactive Streams interfaces
+
+An sometimes overlooked crucial piece of information about [Reactive Streams](https://github.com/reactive-streams/reactive-streams-jvm/) is that they are a [Service Provider Interface](https://en.m.wikipedia.org/wiki/Service_provider_interface), as explained in depth in one of the [early discussions](https://github.com/reactive-streams/reactive-streams-jvm/pull/25) about the specification.
+Akka Streams was designed during the development of Reactive Streams, so they both heavily influenced one another.
+
+It may be sunrising to learn that even within the Reactive Specification the types had initially attempted to hide `Publisher`, `Subscriber` and the other SPI types from users of the API.
+Though since those internal SPI types would end up surfacing to end users of the standard in some cases, it was decided to [remove the API types, and only keep the SPI types][9] which are the `Publisher`, `Subscriber` et al.
```

Comment:
what is that 9?
should it be a full link to an issue?

Reply:
Thanks fixed yes, it was a copy pasta from stack overflow, thanks for noticing